### PR TITLE
fix files tags headers

### PR DESCRIPTION
**Pull request by BazookaJoe1900**
### Solved Problem
many files has wrong file tag on their header description.

### Solution
I wrote script to find and fix the files

### CODE REVIEW

**Comment by BazookaJoe1900** [comment]
- I have only searched for c, cpp, h and hpp files
 - I have only fixed tag look like `@file`
 - I have only fixed the main repo

if there is need to expend the script, tell me

**Comment by dagar** [comment]
How about eliminating them instead? I can't imagine these actually provide value given how wrong they frequently are. 

I would also propose dropping per file author tags as well, that's what git blame is for.

FYI @bresch

**Comment by BazookaJoe1900** [comment]
I actually like the file name on the file it self. when it correct, it helps when you try to search the file references.
meaning, if I want to find where xxx.h is used, it help to see xxx.h too on the search results.

about the authors names, it issue of credit, I thinks.

File: src/drivers/distance_sensor/broadcom/afbrs50/Inc/api/argus_dfm.h [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*************************************************************************//**
- * @file
+ * @file argus_dfm.h
```
[the remark this answers is not in the document]
Comment (BazookaJoe1900) [reply src/drivers/distance_sensor/broadcom/afbrs50/Inc/api/argus_dfm.h:2]: I will remove.
any other places you might know that we should not 

there are many places that there is "@file" but no name. I can skip "fixing" it by adding the name of the file

**Comment by BazookaJoe1900** [comment]
@dagar any reason not to merge it? there is need to change anything?

**Comment by bresch** [comment]
> I would also propose dropping per file author tags as well, that's what git blame is for.

I agree 100% https://youtube.com/clip/UgkxFrPo30esfK5g9chSNaljU-IMqmZrqVUR?si=R5nxZNwefYnWGVja

**Comment by BazookaJoe1900** [comment]
@bresch what do you think about the file name? should we leave them? I prefer to keep, like I said it helps me to when I search for file references, to see the file itself on the search results

**Comment by dakejahl** [comment]
@BazookaJoe1900 if you're using sublime text or vscode you can use ctrl + p to search for file names

**Comment by BazookaJoe1900** [comment]
the original pr meant to fix the errors tags.
I think that merging the current pr is valid, and wider change of the files header can be done on another pr.

I am without a firm opinion about the author tag. but it can be done after decision will be made about 'new' header format.
who should decide that? How about poll?

BTW
I checked the option of changing the authors tags, which is more complicated, because there are different syntax of this, for example:
https://github.com/PX4/PX4-Autopilot/blob/e4dd3f28715b83ee49cf39b3ce68853b3926d52e/platforms/nuttx/NuttX/Make.defs.in#L5 

on some places the name and the 'author' are mixed with the copyright:
https://github.com/PX4/PX4-Autopilot/blob/e4dd3f28715b83ee49cf39b3ce68853b3926d52e/boards/mro/x21/nuttx-config/include/board.h#L5-L6

there are some 'inline' notes:
https://github.com/PX4/PX4-Autopilot/blob/e4dd3f28715b83ee49cf39b3ce68853b3926d52e/src/lib/geo/geo.cpp#L216

I can 'clear' only the basic ones like this one:
https://github.com/PX4/PX4-Autopilot/blob/e4dd3f28715b83ee49cf39b3ce68853b3926d52e/src/modules/mc_att_control/mc_att_control_main.cpp#L38-L45

**Comment by bresch** [comment]
You could leave the file names if you find it useful, I'm fine with that.
Removing the author names starting with `@author` would already be useful, yes. Feel free to do another PR for that.

**Review by bresch — APPROVED** [review]

**Pull request merged**